ehttp/src/types.rs

```diff
@@ -71,7 +71,10 @@ impl Response {
     }
 
     pub fn content_type(&self) -> Option<&str> {
-        self.headers.get("content-type").map(|s| s.as_str())
+        self.headers
+            .iter()
+            .find(|s| s.0 == "content-type")
+            .map(|s| s.1.as_str())
     }
 }
```

Comment:
It would be nice with a helper function here for reading any header

Reply:
Good idea, I've been writing postman-like tools in egui lately, so maybe I can take the opportunity to perfect ehttp!
https://github.com/tmtbe/rust-ui

Reply:
You may need to re-approve it, something hasn't been changed enough to cause a compilation error, thanks!

Reply:
I think you pushed the wrong thing 😬

Reply:
Sorry, I should have submitted it separately, now it's correct.